cmd/eval.go

```diff
@@ -232,6 +234,7 @@ Set the output format with the --format flag.
     --format=pretty    : output query results in a human-readable format
     --format=source    : output partial evaluation results in a source format
     --format=raw       : output the values from query results in a scripting friendly format
+    --format=discard   : output the result field as "discarded"
```

Comment:
Nit: `output the result field as "discarded"` lets clarify that this is the case when the result is non-nil.

Reply:
Does this clarify it?
<img width="596" alt="Screenshot 2023-07-12 at 9 48 18 PM" src="https://github.com/open-policy-agent/opa/assets/50806654/54afcb34-f51b-41db-b4b8-ae56d997b14d">

Reply:
@26tanishabanik sorry I meant lets update the text in the `--format=discard ` description to something like `--format=discard   : output the result field as "discarded" when non-nil`

Reply:
Sure, will update that